README.md

```diff
@@ -364,6 +367,35 @@ Marks certain field indexes as unusable by the smithy specification. For example
 
 For full documentation on what each of these traits does, see the smithy specification [here](modules/core/resources/META-INF/smithy/proto/proto.smithy).
 
+
+### alloy#dateFormat
+
+This trait is used to express that a `String` in your model is formatted as a date.
```

Comment:
Can't remember where we landed on this, but do we want to put an explicit format for the dates here? `yyyy-mm-dd` or whatever?

Reply:
I guess... I was not sure if we wanted to duplicate this info since I added it as documentation on shape itself: https://github.com/disneystreaming/alloy/pull/39

Reply:
I think we should put it in the readme so people can quickly see if it is something that will work for a given use case or not, but that's just my 2 cents so I am fine with whatever you decide